extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php

```diff
@@ -44,9 +44,49 @@ static public function onParserTagHooksBeforeInvoke( $name, $marker, $content, $
 		return true;
 	}
 
-	static public function onWgQueryPages( &$queryPages = [ ] ) {
+	public static function onWgQueryPages( &$queryPages = [ ] ) {
 		$queryPages[] = [ 'AllinfoboxesQueryPage', 'AllInfoboxes' ];
 
 		return true;
 	}
+
+	/**
+	 * @param EditPageLayoutController $editPage
+	 *
+	 * @return bool
+	 */
+	public static function onEditPageLayoutExecute( $editPage ) {
+		// run only on template
+		$requestContext = $editPage->getContext();
+		if ( $requestContext->getTitle()->getNamespace() == NS_TEMPLATE &&
```

Comment:
Can there be a situation in which $requestContext->getTitle() returns a non-object? If Title is null (which as I recall can happen) the code may die here.
Maybe a more rigid check would make sense here?